src/Disbursements.php

```diff
@@ -64,7 +64,7 @@ public static function createReqParams()
      * 'reference'=> string,
      * 'total_uploaded_amount'=> int,
      * 'total_uploaded_count'=> int,
-     * 'status'=> string,
+     * 'status'=> 'NEEDS_APPROVAL',
```

Comment:
Only one status?

Reply:
yes, according to the API Reference

Reply:
Please ask the Disbursements team if they have more status. Because it is weird if they only have one status

Reply:
For create batch disbursements, I think it's awaiting to be approved first to be processed further when it's just recently created. After approved, then the status will be either `COMPLETED`, `CHECK`, `DELETE`, or `FAILED`.

Reply:
Great. Thanks @ellennugroho . Maybe you can update the status in the get disbursements?

Reply:
Never mind, since you use example for the get disbursements